src/main/java/net/sf/jabref/wizard/auximport/AuxFileParser.java

```diff
@@ -274,6 +189,33 @@ private void resolveTags() {
     }
 
     /*
+     * Resolves and adds CrossRef entries
+     */
+    private void resolveCrossReferences(BibEntry entry) {
+        entry.getFieldOptional("crossref").ifPresent(crossref -> {
+            if (!uniqueKeys.contains(crossref)) {
+                BibEntry refEntry = masterDatabase.getEntryByKey(crossref);
+
+                if (refEntry == null) {
+                    unresolvedKeys.add(crossref);
+                } else {
+                    insertEntry(refEntry);
+                    crossRefEntriesCount++;
+                }
+            }
+        });
+    }
+
+    /*
+     * Insert a clone of the given entry. The clone is given a new unique ID.
+     */
+    private void insertEntry(BibEntry entry) {
+        BibEntry clonedEntry = (BibEntry) entry.clone();
+        clonedEntry.setId(IdGenerator.next());
+        auxDatabase.insertEntry(clonedEntry);
+    }
+
+    /*
      *  Copy the database's configuration, i.e., preamble and strings.
      */
     private void copyDatabaseConfiguration() {
```

Comment:
I think it would make sense to move the copyDatabaseConfiguration method to the BibDatabase class (since the db class knows best what to copy).